### DISCUSSION TITLE
Echo Speaks....again

### DISCUSSION THREAD

**Discussion by jallen2281** [tonesto7/echo-speaks#892] [Show and tell]
I just wanted to throw it out there...this software is still very much alive for SmartThings users. I was able to implement a local server and with a couple simple code changes, get it working again just as it was before.

**Comment by AnarchismUK** [comment]
Any help appreciated

**Reply by jallen2281** [reply]
Change the name and point it to your local server. Smartthings has blocked the app by the name, so making it "Echo Speaks Again" or something other than "Echo Speaks" unblocks the app.
